keyboards/phantagom/gigan/readme.md

```diff
@@ -0,0 +1,29 @@
+# phantagom/gigan
+
+[Gigan](https://en.wikipedia.org/wiki/gigan) is a kaiju from Toho's Godzilla franchise who first appeared in Godzilla vs. Gigan.
+
+![phantagom/gigan](https://i.imgur.com/EEJFeOlh.jpeg)
+
+This is a custom handwired numpad running the QMK firmware, which features an additional column for macros. The device is equipped with LEDs to indicate the active layer and an OLED display that can show various information.
```

Comment:
If this is a handwired project it should reside in the `/keyboards/handwired/` directory

Reply:
done